Tests/Adapter/PHPCR/PHPCRAdapterTest.php

```diff
@@ -5,17 +5,16 @@
 use PHPUnit\Framework\TestCase;
 use Vich\UploaderBundle\Adapter\PHPCR\PHPCRAdapter;
 use Vich\UploaderBundle\Tests\DummyEntity;
+use Doctrine\Common\Persistence\Event\LifecycleEventArgs;
```

Comment:
I know :). Sadly `.styleci.yml` file not helps, because configured in drowser settings have higher priority. Will open separate PR and trigger repo owner.